.github/workflows/stale.yaml

```diff
@@ -0,0 +1,21 @@
+# https://github.com/marketplace/actions/stale-branches
```

Comment:
Added this in ... because: 
![image](https://github.com/Nextdoor/k8s-charts/assets/768067/1e72f576-9749-4dc8-a1ae-e19269ebe707)